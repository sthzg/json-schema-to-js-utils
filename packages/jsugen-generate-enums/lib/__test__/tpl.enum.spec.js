import enumTpl from '../enum.tpl';

test('should generate the expected template output', () => {
  const ANY_STRING_VALUE = 'any string';
  const ANY_NUMBER_VALUE = 99;

  const content = enumTpl({
    data: {
      pathName: 'any_name',
    },
    schema: {
      enum: [ANY_STRING_VALUE, ANY_NUMBER_VALUE],
    },
  });

  expect(content).toMatchSnapshot();
});
