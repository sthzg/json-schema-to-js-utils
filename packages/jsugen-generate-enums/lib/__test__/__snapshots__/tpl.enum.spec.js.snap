// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should generate the expected template output 1`] = `
"
export const AnyNameValues = [
  \\"any string\\",
  99
];
export const AnyNameEnum = {
  \\"99\\": 99,
  \\"any string\\": \\"any string\\"
};
"
`;
